_inc/client/at-a-glance/scan.jsx

```diff
@@ -180,13 +184,15 @@ class DashScan extends Component {
 		return (
 			<JetpackBanner
 				callToAction={ __( 'Upgrade', 'jetpack' ) }
-				title={ __( 'Find threats early so we can help fix them fast.', 'jetpack' ) }
+				title={ __(
+					'Automated scanning and one-click fixes keep your site ahead of security threats.',
```

Comment:
Just out of curiosity, where is this copy change coming from? Does it come from the Offer Reset copy document?

Reply:
I've just added the link in the ticket.

Reply:
Thank you!